help_popup/models/action_window.py

```diff
@@ -0,0 +1,179 @@
+# coding: utf-8
+# © 2015 David BEAL @ Akretion <david.beal@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import inspect
+
+from openerp import _, api, models, fields
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from bs4 import BeautifulSoup as BSHTML
+except ImportError:
+    _logger.debug(
```

Comment:
this logger is for not installed module (oca guidelines)
There is raise in standard odoo installation module with 'external_dependencies' data in manifest.
No way to install if no dependency satisfied.